bin/axe-storybook.js

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env node
 
-// eslint-disable-next-line import/extensions
```

Comment:
I removed eslint-plugin-import, because it doesn't support flat config yet and is more complex to setup than most plugins.